lua/nvim-treesitter-playground/internal.lua

```diff
@@ -108,7 +141,12 @@ local function setup_buf(for_buf)
   vim.cmd 'augroup END'
 
   api.nvim_buf_set_keymap(buf, 'n', 'o', string.format(':lua require "nvim-treesitter-playground.internal".toggle_query_editor(%d)<CR>', for_buf), { silent = true })
-  api.nvim_buf_set_keymap(buf, 'n', 'i', string.format(':lua require "nvim-treesitter-playground.internal".toggle_highlights(%d)<CR>', for_buf), { silent = true })
+  api.nvim_buf_set_keymap(buf, 'n', 'h', string.format(':lua require "nvim-treesitter-playground.internal".toggle_hl_groups(%d)<CR>', for_buf), { silent = true })
```

Comment:
This mapping is unfortunate. You can't move to the left when using vim movements.

Reply:
lol didn't realize that when testing it. I'll change it.